app/views/claims/support/schools/mentors/index.html.erb

```diff
@@ -1,17 +1,13 @@
-<% content_for :page_title, "#{@school.name} - #{t(".heading")}" %>
+<% content_for :page_title, t(".page_title") %>
 
-<% render "claims/support/primary_navigation", current: :organisations %>
+<% render "claims/support/primary_navigation", school: @school, current: :mentors %>
```

Comment:
I think you'll need to revert this. 
1. The support nav doesn't have a mentors link.
2. The page title for claims services, still need to include the school name.

![screencapture-track-and-pay-1169-test-teacherservices-cloud-support-schools-fff7c7ed-e9e4-4c2d-833b-e1867987260a-mentors-2024-11-07-16_12_16](https://github.com/user-attachments/assets/6d5a926d-a9ce-4b56-8ab5-88f450825629)